notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>comment about the numbered steps being inconsistent with rest of notebook, suggestion to remove </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Ah it is the standard format in the notebook template.</p>

Reply:
<p>Will keep to be consistent unless we change the template.</p>